app/presenters/flood_risk_engine/application_version_page_presenter.rb

```diff
@@ -6,12 +6,24 @@ def application_name
       FloodRiskEngine.config.application_name || Rails.application.class.module_parent_name
     end
 
-    def application_version
-      defined?(::Application::VERSION) ? Application::VERSION : "Undefined"
-    end
-
     def git_commit
-      @git_commit ||= GitCommitSha.current
+      @_git_commit ||= begin
+        sha =
+          if Rails.env.development?
+            `git rev-parse HEAD`
+          else
+            heroku_file = Rails.root.join ".source_version"
+            capistrano_file = Rails.root.join "REVISION"
+
+            if File.exist? capistrano_file
+              File.open(capistrano_file, &:gets)
+            elsif File.exist? heroku_file
```

Comment:
Are we using heroku?

Reply:
I think it may occasionally get used for testing, but it's not in active use right now.